libs/spark/stories/navbar.stories.tsx

```diff
@@ -26,22 +26,31 @@ const Template = (args: NavBarProps) => (
   <NavBar {...args}>
     <Toolbar>
       <PrendaMonogramDark fontSize="62px" />
-      <NavbarLink href="/">
-        <HomeIconDuotone fillColor="transparent" />
-        <span>Dashboard</span>
-      </NavbarLink>
-      <NavbarLink href="/" aria-current="page">
-        <CheckCircleIconDuotone />
-        <span>Active</span>
-      </NavbarLink>
-      <NavbarLink className="disabled">
-        <RocketIconDuotone />
-        <span>Disabled</span>
-      </NavbarLink>
-      <NavbarLink style={{ marginLeft: 'auto' }}>
-        <InboxFilledIconDuotone />
-        <span>0</span>
-      </NavbarLink>
+      <NavBarItem
+        href={`#`}
+        startIcon={<HomeIconDuotone fillColor="transparent" />}
```

Comment:
> Carry to rest of icons

Having trouble with this one, do we need to do a theme override? 

[https://stackoverflow.com/questions/62677833/change-size-of-starticon-endicon-material-ui]

Reply:
No need for theme overrides, anything we can do there we can do here. I'll note though, I think my theme overrides to the SvgIcon might have been heavy handed and making life more difficult than necessary here in places -- but that's for another time, and not affecting this specific comment directly.

See the stylesheet below. There's a style rule with higher specificity than the rule that gets applied when `fontSize="large"` (i.e. the one setting `font-size: 2rem`). So you just have to match that winning rule to have your change take effect. We want to do that -- fixing intended prop behavior -- rather than being heavy handed in that rule by overriding every instance of an icon to be `2rem`.
![image](https://user-images.githubusercontent.com/25781782/125340313-55085480-e320-11eb-9058-a4a6f1fbb392.png)
So something like
```

  iconSizeMedium: {
    '& > *:first-child': {
      fontSize: '1.5rem', // FYI, our small/medium/large for icons is 16px/24px/32px
      '&.MuiSvgIcon-fontSizeLarge': {
        fontSize: '2rem',
      },
    },
  },
```